models/smi-test.go

```diff
@@ -0,0 +1,27 @@
+package models
```

Comment:
Why not use https://github.com/layer5io/gokit/blob/master/smi/smi.go? Or if we have smi result model here then why is their need for https://github.com/layer5io/gokit/blob/master/smi/smi.go.

Reply:
Because the one in `models/` is for limiting the api parameters that are exposed, and the one given out from `gokit` is all the available parameters from the test. This is done so that the meshery server has the control on which parameters to expose and which not (Easy to make changes).